engine/requirements.txt

```diff
@@ -1,41 +1,40 @@
-django==3.2.19
-djangorestframework==3.12.4
+django==4.0.10
+djangorestframework==3.14.0
 slackclient==1.3.0
 whitenoise==5.3.0
 twilio~=6.37.0
 phonenumbers==8.10.0
-django-ordered-model==3.1.1
-celery[amqp,redis]==5.2.7
-redis==3.4.1
+django-ordered-model==3.5
+celery[amqp,redis]==5.3.0rc2
+redis==4.5.5
 humanize==0.5.1
 uwsgi==2.0.21
-django-cors-headers==3.7.0
-django-debug-toolbar==3.2.1
+django-cors-headers==4.0.0
+django-debug-toolbar==4.0.0
 django-sns-view==0.1.2
 python-telegram-bot==13.13
 django-silk==5.0.3
-django-redis-cache==3.0.0
```

Comment:
see [this comment](https://github.com/grafana/oncall/pull/2079/files#r1213329641). tldr; this library doesn't support `django` v4 + `django` v4 now natively supports redis